pkg/vfs/dirfs.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 Chainguard, Inc.
```

Comment:
Is DirFS something we expect to use in melange or elsewhere? In general I'd bias toward keeping it `internal` until the API settles and we decide we want external folks to use it.

Reply:
Yes, I expect to use it in Melange.